pkg/provider/acme/local_store.go

```diff
@@ -16,25 +15,33 @@ var _ Store = (*LocalStore)(nil)
 
 // LocalStore Stores implementation for local file
 type LocalStore struct {
+	saveDataChan chan map[string]*StoredData
 	filename     string
-	storedData   *StoredData
-	SaveDataChan chan *StoredData `json:"-"`
+	storedData   map[string]*StoredData
 	lock         sync.RWMutex
```

Comment:
The convention is to place the lock above the field(s) that it protects, or at least to document it. As it is, we don't know which one(s) among saveDataChan, filename, and storedData, lock protects.

ex:

type LocalStore struct {
  lock sync.RWMutex
  filename string

  saveDataChan chan map[string]*storedData
  storedData map[string]*StoredData
}

would mean that lock guards filename.